js/Index.js

```diff
@@ -0,0 +1,41 @@
+// create an array for book collection
+// function to add book to collection
+// function to remove book from collection
+const bookCollection = JSON.parse(localStorage.getItem('bookCollection')) || [];
+const display = document.getElementById('display-books');
+const title = document.getElementById('book-title');
+const author = document.getElementById('book-author');
+const btnAdd = document.getElementById('add');
+
+function collection() {
+  display.innerHTML = '';
+  for (let i = 0; i < bookCollection.length; i += 1) {
+    display.innerHTML += `<div class="book">
+        <h3>${bookCollection[i].title}</h3>
+        <p>${bookCollection[i].author}</p>
+        <button class="remove" onclick="remove(${i})">Remove</button>
+        </div>`;
+  }
+}
+collection();
+function remove(index) {
+  bookCollection.splice(index, 1);
+  localStorage.setItem('bookCollection', JSON.stringify(bookCollection));
+  collection();
+}
+
```

Comment:
- [ ] As suggested by previous reviewer and according to the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md), please make sure that the data is preserved is browser's memory. Right now, the data is not correctly preserved in local storage. Please consider fixing this 🎯 I know yo can do that 💪

Reply:
Hello Kenza, We cannot find a solution to this problem can we open a direct meet if it is possible. thank you

Reply:
Hello @zied2112 I can see that your project is approved. Congratulations 🥳 For quick reply, please tag reviewers so that we can receive the notifications 🔔 And good luck for the next milestone. I wish you success. 🥇